src/Libraries/Microsoft.Extensions.AI.Abstractions/AudioTranscription/StreamingAudioTranscriptionUpdate.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>
+/// Represents a single streaming response chunk from an <see cref="IAudioTranscriptionClient"/>.
+/// </summary>
+/// <remarks>
+/// <para><see cref="StreamingAudioTranscriptionUpdate"/> is so named because it represents streaming updates
+/// to an audio transcroption. As such, it is considered erroneous for multiple updates that are part
+/// of the same audio to contain competing values. For example, some updates that are part of
+/// the same audio may have a <see langword="null"/> value, and others may have a non-<see langword="null"/> value,
+/// but all of those with a non-<see langword="null"/> value must have the same value (e.g. <see cref="StreamingAudioTranscriptionUpdate.CompletionId"/>).
+/// </para>
+/// <para>
+/// The relationship between <see cref="AudioTranscriptionCompletion"/> and <see cref="StreamingAudioTranscriptionUpdate"/> is
+/// codified in the <see cref="StreamingAudioTranscriptionUpdateExtensions.ToAudioTranscriptionCompletionAsync"/> and
+/// <see cref="AudioTranscriptionCompletion.ToStreamingAudioTranscriptionUpdates"/>, which enable bidirectional conversions
+/// between the two. Note, however, that the conversion may be slightly lossy, for example if multiple updates
+/// all have different <see cref="StreamingAudioTranscriptionUpdate.RawRepresentation"/> objects whereas there's
+/// only one slot for such an object available in <see cref="AudioTranscriptionCompletion.RawRepresentation"/>.
+/// </para>
+/// </remarks>
+public class StreamingAudioTranscriptionUpdate
```

Comment:
This can just be AudioTranscriptionUpdate; https://github.com/dotnet/extensions/pull/5870 renames StreamingChatCompletionUpdate to just be ChatResponseUpdate.

Reply:
Yes, renamed to `AudioTranscriptionResponseUpdate`.